i18n/md/collections/2023/en/categories/all.md

```diff
@@ -0,0 +1,40 @@
+---
+id: all
+language: en
+---
+
+#### Overall winner: shadcn/ui 🏆
+
+The hottest project of the year is shadcn/ui, a collection of UI components written in React, allowing customizable styling through TailwindCSS.
+
+Built on top of {radix}, a set of headless components providing primitives for concerns like accessibility and keyboard interactions, shadcn/ui distinguishes itself from other popular libraries (such as MUI, ChakraUI, React Spectrum) by not being a downloadable NPM package. Instead, you integrate a specific shadcn/ui component by running a Command Line that installs the underlying dependencies and copies the source code directly into your codebase.
```

Comment:
I found the `a Command Line` part to be a little strange.

My changes in bold below.

```
Built on top of {radix}, a set of headless components providing primitives for concerns like accessibility and keyboard interactions, shadcn/ui distinguishes itself from other popular libraries (such as MUI, ChakraUI, React Spectrum) by not being a downloadable NPM package. **Instead, you integrate the shadcn/ui components through a terminal command that installs the underlying dependencies and copies the component source code directly into your codebase for further modification.**
```